app/build.gradle

```diff
@@ -10,6 +10,8 @@ android {
         targetSdkVersion 19
         versionCode 1
         versionName "1.0"
+
+        buildConfigField "long", "THEKEY_CLIENTID", "7439283632057308L"
```

Comment:
yeah, it's not considered private information. plus anyone could extract it from the published APK anyways